frontend/ui/elements/page_turns.lua

```diff
@@ -141,6 +141,50 @@ When enabled the UI direction for the Table of Contents, Book Map, and Page Brow
     }
 }
 
+if Device:canAnimate() then
+    table.insert(PageTurns.sub_item_table, {
+        text_func = function()
+            local text = _("Page Turn Animations")
+            if G_reader_settings:isTrue("swipe_animations") then
+                text = text .. "   ★"
+            end
+            return text
+        end,
+        enabled_func = function()
+            return Device:canAnimate()
+        end,
+        checked_func = function()
+            return ReaderUI.instance.view.swipe_animations
+        end,
+        callback = function()
+            UIManager:broadcastEvent(Event:new("ToggleSwipeAnimations"))
+        end,
+        hold_callback = function(touchmenu_instance)
+            local swipe_animations = G_reader_settings:isTrue("swipe_animations")
+            local MultiConfirmBox = require("ui/widget/multiconfirmbox")
+            UIManager:show(MultiConfirmBox:new{
+                text = swipe_animations and _("The default (★) for newly opened books is to enable page turn animations.\n\nWould you like to change it?")
+                or _("The default (★) for newly opened books is to disable page turn animations.\n\nWould you like to change it?"),
+                choice1_text_func = function()
```

Comment:
Does it really need to be per-book ? Does it work nice with some books, and is really ugly with others?
Otherwise, it feels like this should be a global setting.
(You didn't allow to have your invert_ui_layout_mirroring per-book :)

Reply:
I still have to try it on Manga/Comics, because I suspect it works *extremely* well there, and will murder ghosting much better than a simple flash ever could on those fast screens ;).

Again, never actually went further than thinking about it, though ;p.

In any case, it's pretty fun playing with it via FBInk at the lowest speed (60 steps, IIRC) ;).